application/borrower/model.py

```diff
@@ -7,6 +7,7 @@ class Borrower(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     token = db.Column(db.String, nullable=False)
+    deed_token = db.Column(db.String, nullable=True)
```

Comment:
We would prefer a non nullable as this is meant to be like a foreign key relationship.